config/makeprops.py

```diff
@@ -27,7 +27,7 @@
     + "\n\n"
     + "// Generated by "
     + progname
-    + " from file %(inputfile)s, "
+    + " from file %(inputFile)s, "
```

Comment:
I don't understand the trailing "s" here.

Reply:
This old style string format in Python

- `%` is the string formatting operator.
- `s` specifies that the value should be formatted as a string.

We should switch to f""" {inputFile}"""

Reply:
I'll clean this up in a following PR